validateStudentRepo.sh

```diff
@@ -37,16 +39,15 @@ function validateAllExercises {
     cd $studentRepo
 
     echo $SEPARATOR
-    sbt "gotoExerciseNr 0" 2>&1 | grep "Moved to"
```

Comment:
Just made a number of modifications.
`ICselectFirstExercise` has been modified as it didn't update the exercises bookmark and it unnecessarily pulled the solution for the first exercise. Instead now:

- It now just sets the exercises bookmark to the first exercise in the course.
- It is renamed to reflect what it actually does: `ICSetBookmarkToFirstExercise`